web/js/map-OpenLayers.js

```diff
@@ -575,6 +631,12 @@ OpenLayers.Control.Click = OpenLayers.Class(OpenLayers.Control, {
             document.getElementById('side-form').style.display = 'block';
         }
         $('#side').hide();
+        // Although it's now hidden, the category filter for the map is still
+        // posted to the server when the report is sent. The name clashes with
+        // the category select element in the report form, which can cause
+        // issues with the wrong/no category being used for the report.
+        // Work around this by renaming the field when it's not shown.
+        $("select#categories").attr("name", "category_filter");
```

Comment:
Would it be nicer to rename the filter categories option permanently to avoid this clash?